src/screens/studentHome/components/StudentSubjectsList/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { CircularProgress, Pagination, Typography } from '@mui/material';
+import {
+  TCompleteSubject,
+  TCompleteSubject as TSubject,
+} from '../../../../service/requests/useGetSubject/types';
+import StudentSubjectsListItem from '../StudentSubjectsListItem';
+import {
+  Container,
+  FallbackContainer,
+  FallbackTypography,
+  ListContainer,
+  PaginationContainer,
+  ProgressContainer,
+} from './styles';
+
+type Props = {
+  isLoading: boolean;
+  page: number;
+  totalPages: number;
+  subjects?: TSubject[];
+  monitorSubject?: TCompleteSubject;
+  userTypeId?: number;
+  handleChangePage(event: React.ChangeEvent<unknown>, page: number): void;
+  handleSubjectClick(id: number): void;
+  handleConfirmSchedule(subject: TSubject): void;
+  handleManageMonitoringClick(): void;
+  handleOpenEnrollModal(id: number): void;
+};
+
+const StudentSubjectsList = ({
+  isLoading,
+  page,
+  totalPages,
+  subjects,
+  userTypeId,
+  monitorSubject,
+  handleChangePage,
+  handleConfirmSchedule,
+  handleSubjectClick,
+  handleManageMonitoringClick,
+  handleOpenEnrollModal,
+}: Props) => {
+  if (isLoading || subjects === undefined) {
+    return (
+      <Container>
+        <ProgressContainer>
+          <CircularProgress color="primary" />
+        </ProgressContainer>
+      </Container>
+    );
+  }
+
+  if (subjects.length === 0) {
```

Comment:
Quando eu não to matriculado em nada mas sou monitor de uma disciplina, a disciplina que monitoro não aparece porque cai nesse caso de fallback. Devia mostrar a disciplina que monitor e o fallback só na lista de matriculados ([figma](https://www.figma.com/file/Mx9uUhyZCRi4lmVTmDUJbK/Novas-Fun%C3%A7%C3%B5es-%7C-Super-Monitoria?type=design&node-id=843-429&mode=design&t=hzP4SgKT30BNn3Eb-0))